src/handling.rs

```diff
@@ -25,6 +25,7 @@ impl MessageHandler {
         mut inbound_receiver: Receiver<MessageBeanIn>,
         outbound_sender: Sender<MessageBeanOut>,
         listener_sender: Sender<Vec<u8>>,
+        auto_broadcast: bool,
```

Comment:
The PR title refers to this param as `auto_propagate`. Which is the correct name?

Reply:
I thought about this as well, so given you've brought it up, I think it makes sense to change it since we both have a slight concern about this 👍